zerver/webhooks/bitbucket3/doc.md

```diff
@@ -0,0 +1,23 @@
+Zulip supports both Git and Mercurial notifications from
+Bitbucket. This integration is for the the new-style Bitbucket
+webhooks used by Bitbucket Server.
```

Comment:
Make sure you replace `Bitbucket` with `BitBucket` in every instance. The `Bucket` should be capitalized.